src/NJsonSchema.CodeGeneration.Tests/TypeScript/DateCodeGenerationTests.cs

```diff
@@ -1,23 +1,24 @@
-﻿using System;
-using System.Threading.Tasks;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
 using NJsonSchema.CodeGeneration.TypeScript;
 
 namespace NJsonSchema.CodeGeneration.Tests.TypeScript
 {
     [TestClass]
     public class DateCodeGenerationTests
     {
-        public class ClassWithDateProperty
-        {
-            public DateTime MyDateTime { get; set; }
-        }
+        private const string Json =
+@"{
+	""type"": ""object"", 
+	""properties"": {
+		""myDate"": ""2017-01-01""
+	}
+}";
 
         [TestMethod]
-        public async Task When_date_handling_is_string_then_string_property_are_generated_in_class()
+        public void When_date_handling_is_string_then_string_property_are_generated_in_class()
         {
             //// Arrange
-            var schema = await JsonSchema4.FromTypeAsync<ClassWithDateProperty>();
+            var schema = JsonSchema4.FromData(Json);

```

Comment:
FromData() is wrong here - FromData() creates a schema from sample data, in this case this would be a schema with the properties "type" and "properties". Either we use "FromJson()" here or we use "FromTypeAsync" (like in the other tests) and add the [JsonSchema("string", Format = "Date")]` attribute

Reply:
I swear this passed locally!
Will change as suggested tomorrow.